documentation/asciidoc/topics/ref_container_storage.adoc

```diff
@@ -1,15 +1,17 @@
 [id='storage_resources-{context}']
 = Storage Resources
 By default, {ispn_operator} allocates `1Gi` for storage for both
-{cacheservice} and {datagridservice} nodes. You can configure storage size for
-{datagridservice} nodes but not {cacheservice} nodes.
+{cacheservice} and {datagridservice} nodes. You can configure storage resources
+for {datagridservice} nodes but not {cacheservice} nodes.
 
 [source,options="nowrap",subs=attributes+]
 ----
 include::yaml_annotated/cr_container_storage.yaml[]
 ----
 
 <1> Configures the storage size for {datagridservice} nodes.
+<2> Defines whether storage is ephemeral or not. Set the value to `true` to use ephemeral storage, which means all data in storage is deleted when clusters shut down or restart. The default value is `false`.
+<3> Specifies the name of a StorageClass object to use for the persistent volume claim.
```

Comment:
@dmvolod Is there any default for the StorageClass? is it standard? what happens if the user omits this or leaves a null value?

Reply:
@oraNod the default Storage Class name depends of the platform where k8s/ocp/okd is deploying.
https://kubernetes.io/docs/concepts/storage/storage-classes
If user set empty or not exists Storage Class name operator will print an error that it's not exists.
If Storage Class name is not provided, the PVC will use storage class which has an annotation storageclass.kubernetes.io/is-default-class set to true.
https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/